hw2/src/main/java/com/solvd/hw2/mybatis/services/ExperimentServiceMB.java

```diff
@@ -0,0 +1,38 @@
+package com.solvd.hw2.mybatis.services;
+
+import java.io.IOException;
+import java.io.InputStream;
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+import com.solvd.hw2.models.Experiment;
+import com.solvd.hw2.mybatis.dao.ExperimentMBDao;
+
+public class ExperimentServiceMB 
+{
+    private static final String SETUP_PATH = "mybatis.xml";
+    private ExperimentMBDao expDao;
+    private InputStream stream;
+    private SqlSessionFactoryBuilder builder;
+    private SqlSessionFactory factory;
```

Comment:
Optional: for these code, you can create a utility class for this purpose, which would eliminate the need to duplicate these for each other class in the future.